<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Farmer_QtClass</class>
 <widget class="QMainWindow" name="Farmer_QtClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Farmer_Qt</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="BigLand_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/BigLand.jpg</normaloff>../../../../Pictures/BigLand.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="MediumLand_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/MediumLand.jpg</normaloff>../../../../Pictures/MediumLand.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SmallLand_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/SmallLand.jpg</normaloff>../../../../Pictures/SmallLand.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Water_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>120</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/water.jpg</normaloff>../../../../Pictures/water.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Sickle_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/sickle.jpg</normaloff>../../../../Pictures/sickle.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Coin_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/Coin.jpg</normaloff>../../../../Pictures/Coin.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Corn_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/Corn.jpg</normaloff>../../../../Pictures/Corn.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Wheat_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/Wheat.jpg</normaloff>../../../../Pictures/Wheat.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Potatoes_pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../Pictures/Potatoes.jpg</normaloff>../../../../Pictures/Potatoes.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="BigLand_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>12</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MediumLand_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="SmallLand_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>68</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Sickle_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>93</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Water_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>123</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Coin_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>154</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Wheat_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>184</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Corn_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>214</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Potatoes_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>244</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Message_label">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>350</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Farmer_Qt.qrc"/>
 </resources>
 <connections/>
</ui>
